region/maridia/inner-pink.json

```diff
@@ -2178,6 +2178,23 @@
                     {"shinespark": {"frames": 8, "excessFrames": 8}}
                   ],
                   "devNote": "This strat is not useful in-room, but can satisfy a strat in the room before with an exit shinespark."
+                },
+                {
+                  "name": "Leave Charged",
+                  "notable": false,
+                  "requires": [
+                    "canShinechargeMovement",
+                    "Gravity",
+                    {"canShineCharge": {
+                      "usedTiles": 33,
+                      "openEnd": 2
+                    }}
+                  ],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 90
```

Comment:
90 is tricky to get.
For the 100-105 I used `canSlowShortCharge`, just to release run after turning blue so I'm not running on the next jump.